src/pages/index.js

```diff
@@ -50,19 +50,19 @@ const pages = [
     component: AboutPage
   },
   {
-    path: '/footprint',
-    name: 'Our Footprint',
+    path: '/impact',
+    name: 'Our Impact',
     sections: [
       {
-        name: 'Our Footprint',
-        link: '#ourfootprint'
+        name: 'Our Impact',
+        link: '#ourimpact'
```

Comment:
Glad you found this one. I was a little worried my nav-dropdown anchor-tag implementation was overly complicated.